src/columns/Interfaces.jl

```diff
@@ -0,0 +1,26 @@
+is_ch_typename(::Val{N})  where {N} = false
```

Comment:
What's the rationale behind calling this `is_ch_typename`? I only know `typename` from C++ where it stands for type parameters for historic reasons. I'd personally prefer `is_ch_type` for this.

Reply:
>What's the rationale behind calling this is_ch_typename
Because I know English much worse than c++ :) I'll fix it now...

Reply:
done in 5227f33